### PR TITLE
#1956 Database search does not work on the Hive Connection Workbench.

### PR DESCRIPTION
**Pull request by YooYoungmo**
### Description
Hive 커넥션을 사용한 워크벤치에서 데이터베이스 검색이 안되는 오류를 수정 했습니다.

**Related Issue** : 

#1956

### How Has This Been Tested?

* JUnit 테스트 코드를 작성하여 테스트 하였습니다.
* UI에서 Hive 커넥션을 만들고 워크벤치를 생성하여 데이터베이스가 검색되는지 확인 했습니다.

### Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist:

- [ ] My code follows the code style of this project. _it will be added soon_
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document. _it will be added soon_
- [X] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by ufoscw** [comment]
run build
deploy to 5

File: discovery-common/src/main/java/app/metatron/discovery/extension/dataconnection/jdbc/accessor/AbstractJdbcDataAccessor.java [hunk 1]
```diff
@@ -213,7 +213,14 @@ public void useDatabase(String catalog, String database) {
       // 1. TABLE_SCHEM String => schema name
       // 2. TABLE_CATALOG String => catalog name (may be null)
       while (rs.next()) {
-        dataBaseNames.add(rs.getString(1));
```
Comment (ufoscw) [inline discovery-common/src/main/java/app/metatron/discovery/extension/dataconnection/jdbc/accessor/AbstractJdbcDataAccessor.java]: @YooYoungmo 
The DatabaseMetadata object has the getSchemas (catalog, schemaPattern) function.
What do you think of the modified code below?
```
      String schemaPatternParam = schemaPattern;
      if(StringUtils.isNotEmpty(schemaPattern)){
        schemaPatternParam = "%" + schemaPattern + "%";
      }
      conn = this.getConnection();
      rs = conn.getMetaData().getSchemas(catalog, schemaPatternParam);
```

Comment (YooYoungmo) [reply discovery-common/src/main/java/app/metatron/discovery/extension/dataconnection/jdbc/accessor/AbstractJdbcDataAccessor.java]: @ufoscw It's a good idea. I changed the code.

**Comment by ufoscw** [comment]
run build
deploy to 5

**Pull request merged**